lib/logstash/filters/aggregate.rb

```diff
@@ -195,6 +195,28 @@ class LogStash::Filters::Aggregate < LogStash::Filters::Base
   # Example value : `"map['sql_duration'] += event['duration']"`
   config :code, :validate => :string, :required => true
 
+
+
+  # This configuration describes the code block that is executed either on timeout 
+  # or in case 'push_previous_map_as_event' is set to true.
+  #
+  # Example value : `"map['sql_duration'] += event['duration']"`
+  config :timeout_code, :validate => :string, :required => false
```

Comment:
I think we could do a better description for this option :

The code to execute to complete timeout event, when 'push_map_as_event_on_timeout' or 'push_previous_map_as_event' is set to true.  
Example value : `"event['tags'] = '_aggregatetimeout'"`